### ISSUE TITLE
Create a Thimble homepage

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.webmaker.org#675]
Our latest design for the Thimble homepage lives here - https://redpen.io/or0f6e3e18912fc780

We need:
* [ ] A final design review & sign-off
* [ ] A video
* [ ] 3 projects
  * Are the ones in the mockup good to roll with? Do we want something that touches on Javascript?
* [ ] 3 related teaching kids
* [ ] To reconcile the two different shades of green between the homepage and the editor

cc @hannahkane

**Comment by hannahkane** [comment]
Working on the teaching kits now. I like the three that are there now, though I think we could replace either Keep Calm or the Movie Poster with a javascript activity. Maybe @mickfuzz or @jgmac1106 would have some ideas about what that activity could be?

**Comment by jgmac1106** [comment]
I can help with the writing the teaching kits but no nothing of javascript.
There are some javascript activities in the gallery.

**Comment by jgmac1106** [comment]
nor spelling. I no nothing of this magic feet either.

**Comment by hannahkane** [comment]
Thanks, @jgmac1106. I'll look around the gallery.

Also, I'd really like to update the existing Makes with comments, which are being spec'ed out in #573, so this is somewhat dependent on that.

**Comment by jgmac1106** [comment]
@chadsansing has a poetry 180 remix that has a written tutorial that we
could use with the #573 format.

**Comment by jgmac1106** [comment]
@hannakane I hope I did this right. I don't have time to finish but I started a teaching kit for Keep Calm: https://github.com/jgmac1106/webmaker-curriculum/blob/gh-pages/WebLiteracyBasics-I/template.html

I can finish it up tonight after work.

**Comment by jgmac1106** [comment]
@hannahkane So I think I forked my first thing from github, changed it and then added it to my own server. Three things I have never done before. Here is my idea for the "Keep Calm" teaching kit. 

I still have to finish the section on CSS and I messed up the <ol> but if we are going with Back to School as the theme for launch this was my idea: http://sandbox.jgregorymcverry.com/webmaker-curriculum-remix/WebLiteracyBasics-I/keepcalm.html
